toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -32,7 +32,7 @@ Optional arguments:
     MODE            build or test. default:"build"
                         In 'test' mode it will use a pre-built mariner chroot image.
                         In 'build' mode it will use the latest published container.
-    VERISION        1.0 or 2.0. default: "2.0"
+    VERISION        2.0 or 3.0. default: "3.0"
```

Comment:
While you're touching this, would you mind fixing the typo in `VERISION`; should be `VERSION` without the first `I`.

Reply:
Ofcourse sir